muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/dialogs/edit-workspace.tsx

```diff
@@ -389,13 +392,23 @@ class OrganizationEditWorkspace extends React.Component<OrganizationEditWorkspac
           this.setState({ selectedStudents: this.props.currentWorkspace.studentsSelect.users, studentsLoaded: true });
         }
 
-        return <DialogRow modifiers="edit-workspace">
-          <AutofillSelector modifier="add-students"
-            loader={this.doStudentSearch}
-            placeholder={this.props.i18n.text.get('plugin.organization.workspaces.editWorkspace.search.students.placeholder')}
-            selectedItems={this.state.selectedStudents} searchItems={allItems} onDelete={this.deleteStudent} onSelect={this.selectStudent} />
+        return <DialogRow>
```

Comment:
Case 1 uses `<div>` as a wrapper but here we are using `<DialogRow>` component. This needs to be unified so we use `<div>`, `<Dialogrow>` or `<form>` as a wrapper element.